
<!DOCTYPE HTML>
<html lang="" >
    <head>
        <meta charset="UTF-8">
        <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
        <title>scDRS Â· GitBook</title>
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="description" content="">
        <meta name="generator" content="GitBook 3.2.3">
        
        
        
    
    <link rel="stylesheet" href="../gitbook/style.css">

    
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-bibtex-cite/style.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-highlight/website.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-search/search.css">
                
            
                
                <link rel="stylesheet" href="../gitbook/gitbook-plugin-fontsettings/website.css">
                
            
        

    

    
        
    
        
    
        
    
        
    
        
    
        
    

        
    
    
    <meta name="HandheldFriendly" content="true"/>
    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="../gitbook/images/apple-touch-icon-precomposed-152.png">
    <link rel="shortcut icon" href="../gitbook/images/favicon.ico" type="image/x-icon">

    
    <link rel="next" href="../appendix.html" />
    
    

    </head>
    <body>
        
<div class="book">
    <div class="book-summary">
        
            
<div id="book-search-input" role="search">
    <input type="text" placeholder="Type to search" />
</div>

            
                <nav role="navigation">
                


<ul class="summary">
    
    

    

    
        
        
    
        <li class="chapter " data-level="1.1" data-path="../">
            
                <a href="../">
            
                    
                    Introduction
            
                </a>
            

            
        </li>
    

    
        
        <li class="header">I Review Articles</li>
        
        
    

    
        
        <li class="header">II Research Articles</li>
        
        
    
        <li class="chapter " data-level="3.1" >
            
                <span>
            
                    
                    Omics-Based
            
                </span>
            

            
            <ul class="articles">
                
    
        <li class="chapter " data-level="3.1.1" data-path="../research/chaffin2022single.html">
            
                <a href="../research/chaffin2022single.html">
            
                    
                    chaffin2022single
            
                </a>
            

            
        </li>
    

            </ul>
            
        </li>
    
        <li class="chapter " data-level="3.2" >
            
                <span>
            
                    
                    Others
            
                </span>
            

            
        </li>
    

    
        
        <li class="header">III Bioinformatic Tools</li>
        
        
    
        <li class="chapter active" data-level="4.1" data-path="zhang2022polygenic.html">
            
                <a href="zhang2022polygenic.html">
            
                    
                    scDRS
            
                </a>
            

            
        </li>
    

    
        
        <li class="divider"></li>
        
        
    
        <li class="chapter " data-level="5.1" data-path="../appendix.html">
            
                <a href="../appendix.html">
            
                    
                    Appendix
            
                </a>
            

            
        </li>
    

    

    <li class="divider"></li>

    <li>
        <a href="https://www.gitbook.com" target="blank" class="gitbook-link">
            Published with GitBook
        </a>
    </li>
</ul>


                </nav>
            
        
    </div>

    <div class="book-body">
        
            <div class="body-inner">
                
                    

<div class="book-header" role="navigation">
    

    <!-- Title -->
    <h1>
        <i class="fa fa-circle-o-notch fa-spin"></i>
        <a href=".." >scDRS</a>
    </h1>
</div>




                    <div class="page-wrapper" tabindex="-1" role="main">
                        <div class="page-inner">
                            
<div id="book-search-results">
    <div class="search-noresults">
    
                                <section class="normal markdown-section">
                                
                                <h3 id="polygenic-enrichment-distinguishes-disease-associations-of-individual-cells-in-single-cell-rna-seq-data">Polygenic enrichment distinguishes disease associations of individual cells in single-cell RNA-seq data</h3>
<hr>
<h5 id="main">Main</h5>
<p>&#x6587;&#x7AE0;&#x94FE;&#x63A5;&#xFF1A;<a href="https://www.nature.com/articles/s41588-022-01167-z" target="_blank">https://www.nature.com/articles/s41588-022-01167-z</a></p>
<p>&#x53D1;&#x8868;&#x65F6;&#x95F4;&#xFF1A;2022/09/01</p>
<p>&#x53D1;&#x8868;&#x671F;&#x520A;&#xFF1A;Nature Genetics</p>
<p>&#x4F5C;&#x8005;&#x4E0E;&#x5355;&#x4F4D;&#xFF1A;Zhang, M.J., Hou, K., Dey, K.K. et al. from Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, MA, USA</p>
<p>Code Availability&#xFF1A;<a href="https://github.com/martinjzhang/scDRS_paper" target="_blank">https://github.com/martinjzhang/scDRS_paper</a>&#x2009;</p>
<p>Data Availability&#xFF1A;<a href="https://figshare.com/projects/Single-cell_Disease_Relevance_Score_scDRS_/118902" target="_blank">https://figshare.com/projects/Single-cell_Disease_Relevance_Score_scDRS_/118902</a></p>
<p>Instructions&#xFF1A; <a href="https://github.com/martinjzhang/scDRS" target="_blank">https://github.com/martinjzhang/scDRS</a></p>
<p>&#x7B80;&#x4ECB;&#xFF1A;</p>
<p>&#x6587;&#x7AE0;&#x4E2D;&#x63D0;&#x5230;&#x7684;scDRS&#x53EF;&#x4EE5;&#x5229;&#x7528;GWAS&#x7684;&#x6570;&#x636E;&#x8BA1;&#x7B97;scRNA-seq&#x6570;&#x636E;&#x4E2D;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x7684;polygenic disease risk&#x3002;</p>
<p>&#x5176;&#x539F;&#x7406;&#x662F;&#x8BA1;&#x7B97;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x662F;&#x5426;&#x8FC7;&#x8868;&#x8FBE;GWAS&#x7ED9;&#x5B9A;&#x7684;&#x4E0E;&#x75BE;&#x75C5;&#x76F8;&#x5173;&#x7684;&#x57FA;&#x56E0;&#x96C6;&#xFF0C;<em>using an appropriately matched empirical null distribution to estimate well-calibrated P values</em>. </p>
<p>&#x4F5C;&#x8005;&#x5728;74&#x4E2A;&#x75BE;&#x75C5;/traits&#x4E0E;16&#x4E2A;&#x5355;&#x7EC6;&#x80DE;RNA-seq&#x6570;&#x636E;&#x4E0A;&#x4F7F;&#x7528;&#x4E86;scDRS&#xFF0C;&#x8861;&#x91CF;&#x4E86;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x4E0E;&#x75BE;&#x75C5;&#x7684;&#x5173;&#x7CFB;&#x4EE5;&#x53CA;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x5185;&#x7684;heterogeneity&#x3002;&#x540C;&#x65F6;&#x4E5F;&#x5E94;&#x7528;&#x5728;&#x4E86;&#x4E0D;&#x540C;&#x7684;species&#xFF08;&#x5C0F;&#x9F20;&#x4E0E;&#x4EBA;&#x7C7B;&#xFF09;&#x4EE5;&#x53CA;&#x4E0D;&#x540C;&#x7684;&#x6D4B;&#x5E8F;&#x5E73;&#x53F0;&#x3002;</p>
<hr>
<h5 id="results">Results</h5>
<h6 id="1-&#x7B97;&#x6CD5;&#x4ECB;&#x7ECD;">1. &#x7B97;&#x6CD5;&#x4ECB;&#x7ECD;</h6>
<p>&#x9996;&#x5148;&#xFF0C;scDRS&#x5229;&#x7528;MAGMA&#x4ECE;GWAS&#x7684;sumamry statistics&#x6587;&#x4EF6;&#x4E2D;&#x6784;&#x5EFA;&#x4E00;&#x4E2A;&#x4E0E;&#x75BE;&#x75C5;&#x76F8;&#x5173;&#x7684;&#x57FA;&#x56E0;&#x96C6;&#xFF08;top 1000 MAGMA genes&#xFF09;&#x3002;</p>
<p>&#x4E4B;&#x540E;&#xFF0C;scDRS&#x91CF;&#x5316;&#x8FD9;&#x4E9B;&#x57FA;&#x56E0;&#x5728;&#x6BCF;&#x4E2A;&#x7EC6;&#x80DE;&#x5F53;&#x4E2D;&#x7684;&#x8868;&#x8FBE;&#xFF0C;&#x4ECE;&#x800C;&#x8BA1;&#x7B97;&#x51FA;&#x4E00;&#x4E2A;<strong>raw disease score</strong>&#x3002;&#x5728;&#x8FD9;&#x4E2A;&#x8FC7;&#x7A0B;&#x4E2D;&#xFF0C;&#x6BCF;&#x4E2A;&#x57FA;&#x56E0;&#x6839;&#x636E;MAGMA&#x7684;z-score&#x88AB;&#x5206;&#x914D;&#x4E0D;&#x540C;&#x7684;<strong>&#x6743;&#x91CD;</strong>&#xFF0C;&#x540C;&#x65F6;&#x6839;&#x636E;&#x5B83;&#x5728;&#x5355;&#x7EC6;&#x80DE;&#x6570;&#x636E;&#x4E2D;&#x7684;gene-specific technical noise&#x4E5F;&#x5206;&#x914D;&#x4E00;&#x4E2A;<strong>&#x76F8;&#x53CD;&#x7684;&#x6743;&#x91CD;</strong>&#xFF08;&#x8FD9;&#x4E2A;&#x8BA1;&#x7B97;&#x662F;&#x6839;&#x636E;&#x5BF9;mean-variance relationship across genes&#x5EFA;&#x6A21;&#x6765;&#x5B9E;&#x73B0;&#x7684;&#xFF09;&#x3002;</p>
<p>&#x4E3A;&#x4E86;&#x8861;&#x91CF;&#x7EDF;&#x8BA1;&#x5B66;&#x4E0A;&#x7684;&#x663E;&#x8457;&#x6027;&#xFF0C;scDRS&#x540C;&#x65F6;&#x8FD8;&#x8BA1;&#x7B97;&#x4E86;1000&#x7EC4;&#x5BF9;&#x5E94;&#x7684;&#x5BF9;&#x7167;&#x7EC4;&#x57FA;&#x56E0;&#x96C6;&#xFF08;&#x4E0E;&#x8981;&#x8BA1;&#x7B97;&#x7684;gene set&#x6709;&#x76F8;&#x540C;&#x7684;&#x5927;&#x5C0F;&#xFF0C;&#x8868;&#x8FBE;&#x5747;&#x503C;&#x4EE5;&#x53CA;&#x65B9;&#x5DEE;&#xFF09;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x7684;<strong>raw control score</strong> at Monte Carlo (MC) samples&#x3002;</p>
<p>&#x6700;&#x540E;&#xFF0C;scDRS&#x5BF9;&#x521A;&#x624D;&#x8BA1;&#x7B97;&#x7684;raw disease score&#x4E0E;raw control score&#x505A;&#x4E00;&#x4E2A;normalization&#xFF0C;&#x518D;&#x8BA1;&#x7B97;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x7684;P-values based on the empirical distribution.</p>
<p>scDRS&#x7684;output&#x4E3A;&#xFF1A;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x7684;&#x4E0E;&#x75BE;&#x75C5;&#x76F8;&#x5173;&#x7684;p&#x503C;&#xFF0C;&#x6807;&#x51C6;&#x5316;&#x540E;&#x7684;disease scores&#xFF0C;&#x4EE5;&#x53CA;1000&#x7EC4;&#x7684;control scores&#x7528;&#x4E8E;&#x4E0B;&#x6E38;&#x7684;&#x5206;&#x6790;</p>
<h6 id="2-&#x4E0B;&#x6E38;&#x5206;&#x6790;">2. &#x4E0B;&#x6E38;&#x5206;&#x6790;</h6>
<p>scDRS&#x7684;&#x4E0B;&#x6E38;&#x5206;&#x6790;&#x5305;&#x62EC;&#xFF1A;</p>
<p>&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x5C42;&#x9762;&#x7684;&#x5206;&#x6790; - &#x5F15;&#x5165;&#x4E4B;&#x524D;&#x5206;&#x6790;&#x4E2D;&#x5B9A;&#x4E49;&#x597D;&#x7684;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x4FE1;&#x606F;&#xFF0C;&#x8BA1;&#x7B97;&#x6BCF;&#x4E2A;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x4E2D;&#x4E0E;&#x75BE;&#x75C5;&#x76F8;&#x5173;&#x7684;&#x5F02;&#x8D28;&#x6027;&#x3002;</p>
<p>&#x5355;&#x7EC6;&#x80DE;&#x5C42;&#x9762;&#x7684;&#x5206;&#x6790; - &#x8054;&#x7CFB;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x7684;&#x53D8;&#x91CF;&#x4E0E;&#x5B83;&#x7684;disease score&#x3002;</p>
<p>&#x57FA;&#x56E0;&#x5C42;&#x9762;&#x7684;&#x5206;&#x6790; - prioritize&#x4E0E;disease score&#x76F8;&#x5173;&#x8054;&#x7684;&#x57FA;&#x56E0;&#x3002;</p>
<h6 id="3-&#x6570;&#x636E;&#x96C6;&#x4ECB;&#x7ECD;">3. &#x6570;&#x636E;&#x96C6;&#x4ECB;&#x7ECD;</h6>
<p>&#x5728;&#x8FD9;&#x7BC7;&#x6587;&#x7AE0;&#x4E2D;&#xFF0C;&#x4F5C;&#x8005;&#x4F7F;&#x7528;&#x4E86;74&#x4E2A;disease/traits (average N=346K) &#x7684;GWAS summary statistic&#x6587;&#x4EF6;&#xFF0C;16&#x4E2A;scRNA-seq/snRNA-seq&#x7684;&#x6570;&#x636E;&#xFF0C;&#x4E00;&#x5171;&#x5305;&#x62EC;&#x6765;&#x81EA;&#x4EBA;&#x6216;&#x8005;&#x9F20;&#x7684;1.3 million (1,300,000)&#x7EC6;&#x80DE;&#x4EE5;&#x53CA;31&#x4E2A;&#x7EC4;&#x7EC7;/&#x5668;&#x5B98;&#x3002;</p>
<p>&#x5171;&#x5305;&#x62EC;&#x4EE5;&#x4E0B;&#x51E0;&#x4E2A;&#x6570;&#x636E;&#x96C6;&#xFF1A;</p>
<ul>
<li>TMS (Tabula Muris Senis) mouse cell atlas<ul>
<li>FACS (fluorescence-activated cell sorting) &#x6570;&#x636E;&#x7528;&#x5728;&#x521D;&#x671F;&#x7684;&#x5206;&#x6790;&#x4E2D;&#xFF0C;&#x5B83;&#x5305;&#x62EC;23&#x79CD;&#x7EC4;&#x7EC7;&#xFF0C;120&#x79CD;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#xFF0C;&#x5E76;&#x5229;&#x7528;SMART-seq2&#x5EFA;&#x5E93;&#x3002;</li>
<li>&#x5176;&#x4ED6; TMS &#x6570;&#x636E;</li>
</ul>
</li>
<li>TS (Tabul Sapiens) human cell atlas</li>
<li>&#x5176;&#x4ED6;&#x7279;&#x5B9A;&#x7EC4;&#x7EC7;&#x7684;&#xFF0C;&#x6709;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x6CE8;&#x91CA;&#x7684;cell atlas</li>
</ul>
<h6 id="3-&#x7A33;&#x5B9A;&#x6027;&#x4E0E;&#x529F;&#x6548;power&#x8BC4;&#x4F30;">3. &#x7A33;&#x5B9A;&#x6027;&#x4E0E;&#x529F;&#x6548;(power)&#x8BC4;&#x4F30;</h6>
<p>&#x4F5C;&#x8005;&#x5C06;scDRS&#x4E0E;&#x5176;&#x4ED6;&#x4E09;&#x4E2A;&#x8861;&#x91CF;&#x57FA;&#x56E0;&#x96C6;&#x5728;&#x5355;&#x7EC6;&#x80DE;&#x7684;score&#x7684;&#x65B9;&#x6CD5;&#x6BD4;&#x8F83;&#xFF1A;Seurat cell-scoring function, Vision and VAM&#xFF0C;&#x5E76;&#x4E14;&#x5C55;&#x793A;&#x4E86;null simulations&#x4E0E;causal siulations&#x7684;&#x7ED3;&#x679C;&#x3002;</p>
<ul>
<li><p>Null simulation</p>
<ul>
<li>&#x65B9;&#x6CD5;&#xFF1A;&#x4ECE;TMS FACS&#x6570;&#x636E;&#x4E2D;&#x9009;&#x62E9;&#x4E86;10,000&#x4E2A;&#x7EC6;&#x80DE;&#xFF0C;&#x5E76;&#x968F;&#x673A;&#x9009;&#x62E9;&#x4E86;1,000&#x4E2A;&#x53EF;&#x80FD;&#x7684;disease genes&#x3002;&#x5229;&#x7528;MAGMA&#x7684;z-score&#xFF0C;&#x4F5C;&#x8005;&#x7ED9;&#x4E88;&#x5404;&#x4E2A;&#x57FA;&#x56E0;&#x6743;&#x91CD;&#x5E76;&#x8BA1;&#x7B97;&#x4E86;scDRS&#x7684;&#x503C;&#x3002;</li>
<li>&#x7ED3;&#x679C;&#xFF1A;scDRS&#x4E0E;Seurat&#x5C55;&#x793A;&#x4E86;well-calibrated P values&#xFF0C;Vision&#x6709;&#x8F7B;&#x5FAE;&#x4E0A;&#x6DA8;&#x7684;type I error[1]&#xFF0C;VAM&#x6709;&#x4E25;&#x91CD;&#x7684;type I error&#x3002;</li>
</ul>
</li>
<li><p>Causal simulation</p>
<ul>
<li>&#x65B9;&#x6CD5;&#xFF1A;&#x968F;&#x673A;&#x9009;&#x62E9;1,000&#x4E2A;causal genes&#xFF0C;&#x968F;&#x673A;&#x9009;&#x62E9;500&#xFF08;from 10,000&#xFF09;&#x4E2A;&#x7EC6;&#x80DE;&#xFF0C;&#x624B;&#x52A8;&#x5C06;&#x8FD9;&#x7FA4;&#x7EC6;&#x80DE;&#x7684;causal genes&#x7684;&#x8868;&#x8FBE;&#x91CF;&#x63D0;&#x9AD8;&#x3002;&#x4E4B;&#x540E;&#x968F;&#x673A;&#x9009;&#x62E9;1,000 &#x53EF;&#x80FD;&#x7684;dsease genes&#xFF08;&#x4E0E;&#x5B9E;&#x9645;&#x7684;causal genes&#x6709;25%&#x7684;overlap&#xFF09;&#x6765;&#x5F53;&#x4F5C;scDRS&#x7684;input&#x3002;&#x5728;&#x5229;&#x7528;&#x4E09;&#x79CD;&#x65B9;&#x6CD5;&#x8BA1;&#x7B97;disease score&#x7684;&#x65F6;&#x5019;&#x6CA1;&#x6709;&#x5305;&#x542B;&#x6743;&#x91CD;&#xFF0C;&#x56E0;&#x4E3A;&#x505A;&#x6D4B;&#x8BD5;&#x6570;&#x636E;&#x96C6;&#x7684;&#x65F6;&#x5019;&#x6CA1;&#x6709;&#x6D89;&#x53CA;&#x5230;&#x5B83;&#x3002;</li>
<li>&#x7ED3;&#x679C;&#xFF1A;scDRS&#x7684;&#x529F;&#x6548;&#x6BD4;Seurat&#x4E0E;Vision&#x66F4;&#x5F3A;&#xFF0C;at FDR&lt;0.1&#x3002;&#x4F5C;&#x8005;&#x8BF4;&#xFF0C;&#x8FD9;&#x53EF;&#x80FD;&#x662F;&#x56E0;&#x4E3A;scDRS&#x5728;&#x8BA1;&#x7B97;&#x80CC;&#x666F;&#x7684;&#x65F6;&#x5019;&#x8FD8;&#x8BA1;&#x7B97;&#x4E86;gene-specific weights&#xFF0C;&#x5373;&#x5BF9;&#x4E8E;technical noise&#x9AD8;&#x7684;&#x57FA;&#x56E0;&#x7ED9;&#x4E88;&#x66F4;&#x4F4E;&#x7684;&#x6743;&#x91CD;&#x3002;</li>
</ul>
</li>
</ul>
<h6 id="4-&#x5B9E;&#x9645;&#x6570;&#x636E;&#x4E2D;&#x7684;&#x5E94;&#x7528;&#x4F8B;&#x5B50;">4. &#x5B9E;&#x9645;&#x6570;&#x636E;&#x4E2D;&#x7684;&#x5E94;&#x7528;&#x4F8B;&#x5B50;</h6>
<p>&#x5206;&#x6790;&#x4E00;&#xFF1A;&#x4F5C;&#x8005;&#x628A;scDRS&#x5E94;&#x7528;&#x4E8E;TMS FACS&#x6570;&#x636E;&#xFF08;&#x5305;&#x542B;120&#x4E2A;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#xFF09; + 74&#x4E2A;GWAS diseases/traits&#x3002;</p>
<p>scDRS command:</p>
<pre><code>scdrs compute-score \
--h5ad_file $H5AD_FILE\
--h5ad_species mouse\
--cov_file $COV_FILE\
--gs_file $GS_FILE\
--gs_species human\
--flag_filter_data True\
--flag_raw_count True\
--n_ctrl 1000\
--flag_return_ctrl_raw_score False\
--flag_return_ctrl_norm_score False\
--out_folder $OUT_FOLDER

revision:
python3 /n/home11/mjzhang/gwas_informed_scRNAseq/scDRS/compute_score.py \
--h5ad_file $H5AD_FILE\
--h5ad_species mouse\
--cov_file $COV_FILE\
--gs_file $GS_FILE\
--gs_species human\
--flag_filter True\
--flag_raw_count True\
--n_ctrl 1000\
--flag_return_ctrl_raw_score False\
--flag_return_ctrl_norm_score True\
--out_folder $OUT_FOLDER
</code></pre><p>TS single-cell file process:</p>
<pre><code># Get raw cell counts
adata_full = sc.read_h5ad(DATA_FILE) 
adata_full.X = adata_full.raw.X 
...

# Add conbination metadata
adata_full.obs[&apos;tissue&apos;] = adata_full.obs[&apos;organ_tissue&apos;]
adata_full.obs[&apos;tissue_celltype&apos;] = [&apos;%s.%s&apos;%(x,y) for x,y in zip(adata_full.obs[&apos;tissue&apos;], adata_full.obs[&apos;cell_ontology_class&apos;])] 
...

# Basic QC
sc.pp.filter_cells(adata, min_genes=250)
sc.pp.filter_genes(adata, min_cells=50)
...

# Make cov file
df_cov = pd.DataFrame(index=adata_raw.obs.index)
df_cov[&apos;const&apos;] = 1
df_cov[&apos;n_genes&apos;] = (adata_raw.X&gt;0).sum(axis=1)
df_cov.to_csv(DATA_PATH+&apos;/ts_10X.cov&apos;, sep=&apos;\t&apos;)
...
</code></pre><p><a href="https://www.nature.com/articles/s41588-022-01167-z/figures/3" target="_blank">Figure 3: Disease associations at the cell-type level</a></p>
<p>&#x4E0A;&#x9762;&#x7684;&#x56FE;&#x7247;&#x5C55;&#x793A;&#x4E86;&#x5176;&#x4E2D;&#x6709;&#x4EE3;&#x8868;&#x6027;&#x7684;19&#x4E2A;&#x7EC6;&#x80DE;&#x7C7B;&#x578B;&#x4E0E;22&#x4E2A;disease&#x3002;&#x56FE;&#x4E2D;&#x65B9;&#x5757;&#x8868;&#x793A;&#x4E0E;disease&#x6709;&#x5173;&#x8FDE;&#x7684;celltype(FDR &lt; 0.05), &#x53C9;&#x53F7;&#x8868;&#x793A;disease&#x76F8;&#x5173;&#x7684;&#x5F02;&#x8D28;&#x6027;&#x663E;&#x8457;&#x7684;celltype(FDR &lt; 0.05)&#x3002;</p>
<p>&#x5206;&#x6790;&#x4E8C;&#xFF1A;&#x4F5C;&#x8005;&#x5206;&#x6790;&#x4E86;TMS FACS&#x6570;&#x636E;&#x4E2D;T&#x7EC6;&#x80DE;&#xFF08;3,769&#x4E2A;&#x7EC6;&#x80DE;&#xFF0C;15&#x4E2A;&#x7EC4;&#x7EC7;&#xFF09;&#x7684;&#x5F02;&#x8D28;&#x6027;&#xFF0C;&#x4F7F;&#x7528;&#x4E86;10&#x4E2A;autoimmune disease&#x7684;GWAS&#x6570;&#x636E;&#xFF0C;&#x5E76;&#x4EE5;Heighe&#x505A;&#x9634;&#x6027;&#x5BF9;&#x7167;&#x7684;trait&#x3002;</p>
<p><a href="https://www.nature.com/articles/s41588-022-01167-z/figures/4" target="_blank">Figure 4: Associations of T cells with autoimmune diseases</a></p>
<p>&#xFF08;&#x5355;&#x7EC6;&#x80DE;&#x5C42;&#x9762;&#x7684;&#x5206;&#x6790;&#xFF09;</p>
<p>&#x4F5C;&#x8005;&#x5173;&#x6CE8;&#x4E86;&#x5355;&#x4E2A;&#x7EC6;&#x80DE;&#x4E0E;IBD&#x7684;&#x5173;&#x8054;&#xFF0C;&#x786E;&#x5B9A;&#x4E86;387&#x4E2A;&#x4E0E;IBD&#x76F8;&#x5173;&#x8054;(FDR &lt; 0.1)&#x7684;T&#x7EC6;&#x80DE;&#xFF0C;&#x5206;&#x5E03;&#x5728;4&#x4E2A;T&#x7EC6;&#x80DE;cluster&#x4E2D;(out of 11)&#x3002;&#x5176;&#x4E2D;&#x6709;123&#x4E2A;&#x4E0E;IBD&#x76F8;&#x5173;&#x8054;&#x7684;&#x7EC6;&#x80DE;&#x662F;&#x5728;cluster3&#xFF08;Treg-like&#xFF09;&#x7684;T&#x7EC6;&#x80DE;&#x4E2D;&#xFF0C;&#x8FD9;&#x4E9B;&#x7EC6;&#x80DE;&#x9AD8;&#x8868;&#x8FBE;Treg&#x7684;marker gene(FOXP3, CTLA4, LAG3)&#xFF0C;&#x540C;&#x65F6;&#x4ED6;&#x4EEC;&#x7684;marker&#x57FA;&#x56E0;&#x4E0E;Treg signature&#x6709;&#x663E;&#x8457;&#x7684;overlap&#xFF0C;&#x5E76;&#x4E14;&#x975E;&#x968F;&#x673A;&#x5730;&#x5206;&#x5E03;&#x5728;UMAP&#x56FE;&#x4E0A;&#x3002;</p>
<p> &#x8861;&#x91CF;gene signature&#x7684;&#x76F8;&#x4F3C;&#x6027;&#xFF1A;</p>
<pre><code>MsigDB signatures and effector-like Treg program, two-sided Fisher&apos;s exact test&#x3002;
</code></pre><p>&#x8861;&#x91CF;disease score &#x5728;UMAP&#x56FE;&#x7684;&#x968F;&#x673A;&#x5206;&#x5E03;&#x4E0E;&#x5426;&#xFF1A;</p>
<pre><code>P&#x2009;&lt;&#x2009;0.001, MC test;
</code></pre><p>&#x4F5C;&#x8005;&#x63D0;&#x53D6;&#x4E86;&#x8FD9;&#x4E9B;&#x4E0E;IBD&#x76F8;&#x5173;&#x8054;&#x7EC6;&#x80DE;&#x5BCC;&#x96C6;&#x7684;&#x57FA;&#x56E0;&#xFF08;&#x4E0E;&#x76F8;&#x540C;cluster&#x4E2D;&#x975E;IBD&#x5173;&#x8054;&#x7684;&#x7EC6;&#x80DE;&#x505A;&#x5DEE;&#x5F02;&#x5206;&#x6790;&#xFF09;&#xFF0C;&#x53D1;&#x73B0;&#x8179;&#x808C;&#x4E86;NFkB signaling&#x4EE5;&#x53CA;Th cell&#x5206;&#x5316;&#xFF0C;TNF-mediated signaling&#x7B49;&#x901A;&#x8DEF;&#x3002;</p>
<p>T&#x7EC6;&#x80DE;&#x7279;&#x5F81;&#x57FA;&#x56E0;&#x96C6;&#x7684;&#x641C;&#x96C6;&#xFF1A;</p>
<pre><code>We used MSigDB71 (v7.1) to curate T cell signature gene sets, including naive CD4, memory CD4, effector CD4, naive CD8, memory CD8, effector CD8, Treg, Th1 (T helper 1), Th2 (T helper 2) and Th17 (T helper 17) signatures. For each T cell signature gene set, we identified a set of relevant MSigDB gene sets (22&#x2013;34 gene sets; Supplementary Table 17), followed by selecting the top 100 most frequent genes in these MSigDB gene sets as the T cell signature genes; a gene was required to appear at least twice and genes appearing the same number of times were all included, resulting in 62 to 513 genes for the 10 T cell signature gene sets (Supplementary Table 24). 
</code></pre><p>&#x5C0F;&#x7EC6;&#x80DE;&#x4E9A;&#x7FA4;&#x5BCC;&#x96C6;&#x7684;pathway&#xFF1A;</p>
<pre><code>Pathways enriched in the top 300 specifically expressed genes of the IBD-associated cells and non-associated cells in cluster 3 respectively. The color and number in each cell represent the log10 enrichment p-values (based on Enrichr implemented in GSEAPY124, 125; the implementation used an one-sided Fisher&#x2019;s exact test).
</code></pre><p>&#x540C;&#x6837;&#xFF0C;&#x5BF9;&#x4E8E;&#x5176;&#x4ED6;cluster&#x4E2D;&#x53D1;&#x73B0;&#x7684;&#x4E0E;IBD&#x76F8;&#x5173;&#x8054;&#x7684;&#x7EC6;&#x80DE;&#x505A;&#x4E86;&#x76F8;&#x540C;&#x7684;&#x5206;&#x6790;&#x3002;</p>
<p>&#x4F5C;&#x8005;&#x6BD4;&#x8F83;&#x4E86;T cell effectorness gradient&#xFF08;&#x5229;&#x7528;pseudotime&#x8BA1;&#x7B97;CD4&#x4E0E;CD8&#x7684;effectorness gradient&#xFF09;&#x4E0E;disease score&#xFF0C;&#x53D1;&#x73B0;effectorness&#x4E0E;&#x67D0;&#x4E9B;&#x81EA;&#x8EAB;&#x514D;&#x75AB;&#x75C5;&#x662F;&#x76F8;&#x5173;&#x8054;&#x7684;&#xFF08;P &lt; 0.005, MC test&#xFF09;.</p>
<p>&#xFF08;&#x57FA;&#x56E0;&#x5C42;&#x9762;&#x7684;&#x5206;&#x6790;&#xFF09;
&#x6700;&#x540E;&#xFF0C;&#x4F5C;&#x8005;prioritize&#x4E86;&#x4E0E;&#x75BE;&#x75C5;&#x76F8;&#x5173;&#x8054;&#x7684;&#x57FA;&#x56E0;&#x3002;&#x9488;&#x5BF9;&#x6240;&#x6709;&#x7684;TMS FACS&#x7EC6;&#x80DE;&#xFF0C;&#x4F5C;&#x8005;&#x8BA1;&#x7B97;&#x4E86;&#x7279;&#x5B9A;&#x57FA;&#x56E0;&#x7684;&#x8868;&#x8FBE;&#x4E0E;&#x67D0;&#x4E2A;&#x75BE;&#x75C5;&#x7684;disease score&#x7684;correlation&#xFF0C;&#x4ECE;&#x800C;&#x5F97;&#x5230;&#x4E86;&#x4E0E;GWAS&#x7684;&#x57FA;&#x56E0;&#x5171;&#x8868;&#x8FBE;&#x7684;&#x57FA;&#x56E0;&#x3002;&#x4E0E;&#x5DF2;&#x77E5;&#x7684;Mendelian form of disease&#x57FA;&#x56E0;&#x4E0E;&#x53EF;&#x80FD;&#x7684;&#x836F;&#x7269;&#x9776;&#x70B9;&#x4E0E;&#x505A;&#x6BD4;&#x8F83;&#xFF0C;&#x4F5C;&#x8005;&#x53D1;&#x73B0;&#x76F8;&#x6BD4;&#x4E8E;MAGMA&#xFF0C;&#x901A;&#x8FC7;scDRS&#x80FD;&#x66F4;&#x597D;&#x7684;&#x53D1;&#x73B0;&#x8FD9;&#x4E9B;&#x57FA;&#x56E0;&#x3002;</p>
<p>&#x5206;&#x6790;&#x4E09;&#xFF1A;&#x4F5C;&#x8005;&#x5206;&#x6790;&#x4E86;&#x795E;&#x7ECF;&#x7EC6;&#x80DE;&#x7684;&#x5F02;&#x8D28;&#x6027;&#x4E0E;&#x8111;&#x90E8;&#x76F8;&#x5173;&#x7684;&#x75BE;&#x75C5;&#x7684;&#x5173;&#x7CFB;&#x3002;</p>
<p><a href="https://www.nature.com/articles/s41588-022-01167-z/figures/5" target="_blank">Figure 5: Associations of neurons with brain-related disease/traits and hepatocytes with metabolic traits</a></p>
<p>&#x8861;&#x91CF;&#x4E00;&#x4E2A;cell-level&#x6307;&#x6807;&#x4E0E;disease score&#x7684;&#x5173;&#x7CFB;&#xFF1A;</p>
<pre><code>&#x56FE;&#xFF1A;
  &#x6A2A;&#x8F74;&#x4E3A;&#x8BE5;score quantile bins, &#x7EB5;&#x8F74;&#x4E3A;disease score&#x3002;
&#x8BA1;&#x7B97;&#xFF1A;
  Pearson&#x2019;s correlation between the cell-level variable and the disease score&#x3002;
  For jointly associating multiple cell-level variables with disease, we use the regression t-statistic as the test statistic, obtained from jointly regressing the disease score against the cell-level variables.
</code></pre><p>&#x5206;&#x6790;&#x56DB;&#xFF1A;&#x5E72;&#x7EC6;&#x80DE;&#x4E9A;&#x7FA4;&#x5F02;&#x8D28;&#x6027;&#x4E0E;&#x4EE3;&#x8C22;&#x76F8;&#x5173;traits&#x7684;&#x5173;&#x7CFB;</p>
<p>&#x8BA8;&#x8BBA;TMS FACS&#x7684;&#x809D;&#x7EC6;&#x80DE;(hepatocytes)&#x4E0E;metabolic traits&#x7684;&#x5173;&#x7CFB;&#x3002;</p>
<p>metabolic traits include:</p>
<pre><code>Total cholesterol (TG), high-density lipoprotein (HDL), low-density lipoprotein (LDL), total cholesterol (TC), testosterone (TST), alanine aminotransferase (ALT), alkaline phosphatase (ALP), sex hormone-binding globulin (SHBG) and total bilirubin (TBIL)
</code></pre><hr>
<p>[1]  Type I error&#xFF1A;false-positive, rejects a null hypothesis that is actually true. Type II error is false-negative, if the investigator fails to reject a null hypothesis that is actually false in the population.</p>

                                
                                </section>
                            
    </div>
    <div class="search-results">
        <div class="has-results">
            
            <h1 class="search-results-title"><span class='search-results-count'></span> results matching "<span class='search-query'></span>"</h1>
            <ul class="search-results-list"></ul>
            
        </div>
        <div class="no-results">
            
            <h1 class="search-results-title">No results matching "<span class='search-query'></span>"</h1>
            
        </div>
    </div>
</div>

                        </div>
                    </div>
                
            </div>

            
                
                
                <a href="../appendix.html" class="navigation navigation-next navigation-unique" aria-label="Next page: Appendix">
                    <i class="fa fa-angle-right"></i>
                </a>
                
            
        
    </div>

    <script>
        var gitbook = gitbook || [];
        gitbook.push(function() {
            gitbook.page.hasChanged({"page":{"title":"scDRS","level":"4.1","depth":1,"next":{"title":"Appendix","level":"5.1","depth":1,"path":"appendix.md","ref":"appendix.md","articles":[]},"previous":{"title":"Others","level":"3.2","depth":1,"ref":"","articles":[]},"dir":"ltr"},"config":{"plugins":["bibtex-cite","livereload"],"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"pluginsConfig":{"livereload":{},"search":{},"lunr":{"maxIndexSize":1000000,"ignoreSpecialCharacters":false},"hide-element":{"elements":[".gitbook-link"]},"fontsettings":{"theme":"white","family":"sans","size":2},"highlight":{},"bibtex-cite":{},"sharing":{"facebook":true,"twitter":true,"google":false,"weibo":false,"instapaper":false,"vk":false,"all":["facebook","google","twitter","weibo","instapaper"]},"theme-default":{"styles":{"website":"styles/website.css","pdf":"styles/pdf.css","epub":"styles/epub.css","mobi":"styles/mobi.css","ebook":"styles/ebook.css","print":"styles/print.css"},"showLevel":false}},"theme":"default","pdf":{"pageNumbers":true,"fontSize":12,"fontFamily":"Arial","paperSize":"a4","chapterMark":"pagebreak","pageBreaksBefore":"/","margin":{"right":62,"left":62,"top":56,"bottom":56}},"structure":{"langs":"LANGS.md","readme":"README.md","glossary":"GLOSSARY.md","summary":"SUMMARY.md"},"bibCount":0,"variables":{},"bib":[{"citationKey":"CHAFFIN2022SINGLE","entryType":"ARTICLE","entryTags":{"TITLE":"Single-nucleus profiling of human dilated and hypertrophic cardiomyopathy","AUTHOR":"Chaffin, Mark and Papangeli, Irinna and Simonson, Bridget and Akkad, Amer-Denis and Hill, Matthew C and Arduini, Alessandro and Fleming, Stephen J and Melanson, Michelle and Hayat, Sikander and Kost-Alimova, Maria and others","JOURNAL":"Nature","VOLUME":"608","NUMBER":"7921","PAGES":"174--180","YEAR":"2022","PUBLISHER":"Nature Publishing Group"}},{"citationKey":"ZHANG2022POLYGENIC","entryType":"TECHREPORT","entryTags":{"TITLE":"Polygenic enrichment distinguishes disease associations of individual cells in single-cell RNA-seq data","AUTHOR":"Zhang, Martin Jinye and Hou, Kangcheng and Dey, Kushal K and Sakaue, Saori and Jagadeesh, Karthik A and Weinand, Kathryn and Taychameekiatchai, Aris and Rao, Poorvi and Pisco, Angela Oliveira and Zou, James and others","YEAR":"2022","INSTITUTION":"Nature Publishing Group"}}],"gitbook":"*","theme-default":{"showLevel":true}},"file":{"path":"tools/zhang2022polygenic.md","mtime":"2022-10-14T11:36:17.877Z","type":"markdown"},"gitbook":{"version":"3.2.3","time":"2022-10-14T11:36:27.549Z"},"basePath":"..","book":{"language":""}});
        });
    </script>
</div>

        
    <script src="../gitbook/gitbook.js"></script>
    <script src="../gitbook/theme.js"></script>
    
        
        <script src="../gitbook/gitbook-plugin-livereload/plugin.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search-engine.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-search/search.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/lunr.min.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-lunr/search-lunr.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-sharing/buttons.js"></script>
        
    
        
        <script src="../gitbook/gitbook-plugin-fontsettings/fontsettings.js"></script>
        
    

    </body>
</html>

